modules/UI/UI.js

```diff
@@ -255,26 +255,38 @@ UI.showLocalConnectionInterrupted = function(isInterrupted) {
 
 /**
  * Sets the "raised hand" status for a participant.
+ *
+ * @param {string} id - The id of the participant whose raised hand UI should
+ * be updated.
+ * @param {string} name - The name of the participant with the raised hand
+ * update.
+ * @param {boolean} raisedHandStatus - Whether the participant's hand is raised
+ * or not.
+ * @returns {void}
  */
-UI.setRaisedHandStatus = (participant, raisedHandStatus) => {
-    VideoLayout.setRaisedHandStatus(participant.getId(), raisedHandStatus);
+UI.setRaisedHandStatus = (id, name, raisedHandStatus) => {
+    VideoLayout.setRaisedHandStatus(id, raisedHandStatus);
     if (raisedHandStatus) {
         messageHandler.participantNotification(
-            participant.getDisplayName(),
+            name,
             'notify.somebody',
             'connected',
             'notify.raisedHand');
     }
 };
 
 /**
- * Sets the local "raised hand" status.
+ * Sets the local "raised hand" status and notifies interested listeners that
+ * the raise hand property has changed.
+ * @param {boolean} raisedHandStatus indicates the current state of the
+ * "raised hand"
  */
-UI.setLocalRaisedHandStatus
-    = raisedHandStatus =>
-        VideoLayout.setRaisedHandStatus(
-            APP.conference.getMyUserId(),
-            raisedHandStatus);
+UI.setLocalRaisedHandStatus = raisedHandStatus => {
+    eventEmitter.emit(UIEvents.LOCAL_RAISE_HAND_CHANGED, raisedHandStatus);
```

Comment:
This event is needed to continue supporting the current web toolbox, which relies on pubsub to update some of its state.